CHANGES.md

```diff
@@ -13,12 +13,20 @@ To be released.
  -  Removed `IEquatable<string>` and `IComparable<string>` from `Text`.
     [[#104], [#108]]
  -  Removed `CompositeComparer<TA, TB>` struct.  [[#109]]
+ -  Removed `IEquatable<byte[]>`, `IEquatable<ImmutableArray<byte>>`,
+    `IComparable<byte[]>`, and `IComparable<ImmutableArray<byte>>` from
+    `Integer`.  [[#104], [#110]]
+ -  Removed `IComparable<byte[]>` and `IComparable<ReadOnlyList<byte>>` from
+    `ByteArrayComparer`.  [[#110]]
+ -  Changed the behaviors of `Binary.Equals()` and `Binary.CompareTo()`
```

Comment:
Well, the #106 issue was for changing all types of equality behaviors. I understood it as changing `Integer`'s one only. If not, the issue should be reopened as the title should be updated.

Reply:
Sure, I've reopened the issue and slightly added more wording.